### ISSUE TITLE
Ruby Gem version fails with 2.4.5

### ISSUE THREAD

**Issue by pivotal-anthony-rios** [linuxacademy/content-hello-ruby#1]
Unable to compile buildpack, fails at Ruby installation due to 2.4.5 not being found in rubygems.org

I suggest either removing the Ruby line in the gemfile for stability purposes (ie by default pull the latest of ruby), or use a different source for Ruby.

**Comment by sagogine** [comment]
> Unable to compile buildpack, fails at Ruby installation due to 2.4.5 not being found in rubygems.org
> 
> I suggest either removing the Ruby line in the gemfile for stability purposes (ie by default pull the latest of ruby), or use a different source for Ruby.

Yes, fails to compile the buildpack. Defaults the version to 2.4.9 if the we remove ruby version in gemfile
